File: Logic & Design/Visual Studio Code/Object Oriented Programming/Java/CellPhoneConstructor.java
/*
 * Programmer:      Corey Jenkins
 * Date:            February 29, 2024
 * Filename:        CellPhoneConstructor.java
 * Purpose:         Object-Oriented Programming with a Constructor.
*/

public class CellPhoneConstructor {
    private String manufacturer;
    private String modelNumber;
    private double retailPrice;

    public CellPhoneConstructor(String manufact, String modNum, double retail) {
        manufacturer = manufact;
        modelNumber = modNum;
        retailPrice = retail;
    }

    // Setters or Mutators
    public void setManufacturer(String manufact) {
        manufacturer = manufact;
    }

    public void setModelNumber(String modNum) {
        modelNumber = modNum;
    }

    public void setRetailPrice(double retail) {
        retailPrice = retail;
    }

    public String getManufacturer() {
        return manufacturer;
    }

    public String getModelNumber() {
        return modelNumber;
    }

    public double getRetailPrice() {
        return retailPrice;
    }

    public static void main(String[] args) {
        CellPhoneConstructor myPhone;

        myPhone = new CellPhoneConstructor("Motorolla", "M1000", 199.99);

        System.out.println("The manufacturer is: " + myPhone.getManufacturer());
        System.out.println("The model number is: " + myPhone.getModelNumber());
        System.out.println("The  retail price is: " + myPhone.getRetailPrice());

    }

}

File: Logic & Design/Visual Studio Code/Object Oriented Programming/Python/CellPhoneConstructor.py
'''
Programmer:     Corey Jenkins
Date:           February 29, 2024
Filename:       CellPhoneConstructor.py
Purpose:        Object-Oriented Programming with Constructor.
'''
class CellPhoneConstructor:
    def __init__(self, manufact, model, retail):
        self.manufacturer = manufact
        self.model_number = model
        self.retail_price = retail
        
    def set_manufacturer(self, manufact):
        self.manufacturer = manufact
        
    def set_model_number(self, model):
        self.model_number = model
        
    def set_retail_price(self, retail):
        self.retail_price = retail
        
    def get_manufacturer(self):
        return self.manufacturer
    
    def get_model_number(self):
        return self.model_number
    
    def get_retail_price(self):
        return self.retail_price
    
def main():
    my_phone = CellPhoneConstructor("Motorolla", "M1000", 199.99)
    
    print("The manufacturer is:", my_phone.get_manufacturer())
    print("The model number is:", my_phone.get_model_number())
    print("The retail price is:", my_phone.get_retail_price())
        
main()